src/main/java/Cars.java

```diff
@@ -16,4 +17,31 @@ public void move() {
             car.move();
         }
     }
+
+    public List<Car> findWinners() {
+        return findCarsByPosition(findMaxPosition());
+    }
+
+    public int getCount() {
+        return this.value.size();
+    }
+
+    private int findMaxPosition() {
+        int maxPosition = 0;
+        for (Car car : this.value) {
+            maxPosition = Math.max(maxPosition, car.getPosition());
```

Comment:
```suggestion
            maxPosition = car.max(maxPosition);
```
위와 같이 Car에 메시지를 보내도록 구현하는 것은 어떨까?